lib/vc.js

```diff
@@ -221,10 +221,22 @@ async function verifyCredential(options = {}) {
     }
     return _verifyCredential(options);
   } catch(error) {
+    const result_array = [];
```

Comment:
Couple of comments here. One, please rename `result_array` to `log`.
Also, it shouldn't be set in the `catch()` clause here, it should be returned from `_verifyCredential()`.